<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DeputyDJ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="tabRequests">
       <attribute name="title">
        <string>Track Requests</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </widget>
      <widget class="QWidget" name="tabHistory">
       <attribute name="title">
        <string>Track History</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDJ_Sidekick">
    <property name="title">
     <string>DeputyDJ</string>
    </property>
    <addaction name="actionOpenLog"/>
    <addaction name="separator"/>
    <addaction name="actionLogoutWhatCD"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettingsRequests"/>
    <addaction name="actionHistory"/>
   </widget>
   <addaction name="menuDJ_Sidekick"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSettingRequests">
   <property name="text">
    <string>Requests</string>
   </property>
  </action>
  <action name="actionSettingHistory">
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpenLog">
   <property name="text">
    <string>Open log directory</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettingsRequests">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Requests</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionLogoutWhatCD">
   <property name="text">
    <string>Logout What.CD</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
